invenio_previewer/__init__.py

```diff
@@ -124,7 +124,21 @@
 Creating a record
 ~~~~~~~~~~~~~~~~~
 
-Let's start by creating a record and its persistent identifier:
+When creating a record, the default behaviour is that the class creates
```

Comment:
May I suggest changing "we" to "you"?
And what do you think about this way of phrasing it?
"by default a bucket is created and associated to the record"

Reply:
I'll keep it as we for now as the rest of the documentation uses we